index.html

```diff
@@ -178,14 +178,67 @@ <h2>Tonic</h2>
                   <li><a href="" class="base-link-text3">javascript</a></li>
                 </ul>
               </div>
-              <div class="base-button">
-                <a href="#">See Project</a>
+              <div id = "5" class="base-button">
+                <button class = "btn">See Projects</button>
               </div>
             </div>
           </div>
           <!---End of desktop card 1 Mark up-->
 
-         
+          <!----pop up view--->
+          <div class = "popup-view-bg">
+            <div class = "popup-view">
+            <div class = "top-bar"> 
+            <h3 id = "pop-header">Tonic</h3>
+            <div class = "pop-up-close-button">
+              <span class = "stripe"></span>
+              <span class = "stripe"></span>
+              <span class = "stripe"></span>
+            </div>
+            </div>
+            <div class="popup-list">
+              <ul>
+                <li><a href="" class="text">CANOPY</a></li>
+                <li><img src="images/c1.png" alt="dot icon" /></li>
+                <li><a href="" class="text-2">Back End Dev</a></li>
+                <li><img src="images/c1.png" alt="dot icon" /></li>
+                <li><a href="" class="text-3">2015</a></li>
+              </ul>
+              </div>
+              <div class = "popup-image-wrapper">
+                <img id = "mobile-image" class = "for-mobile" src = "images/section-2.png" alt = "Image of section card"/>
+                <img id = "desktop-image" class = "for-desktop" src = "images/desktop-section-1.png" alt = "Image of section card"/>
+              </div>
+              <div class = "popup-context">
+                <div>            
+            <p>
+              Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the lea</p>
+                </div>
+                <div class = "flex-right">
+              <div>
+                <ul class = "popup-links" id = "base-uls-mobile">
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+                 
+                </ul>
+              </div>
+              <div class = "d-link">
+                <ul id = "base-uls-dt" class = "popup-links">
+                <li class="base-link-text1">github</li>
+                <li class="base-link-text2">ruby</li>
+                <li class="base-link-text3">bootstraps</li>
+                </ul>
```

Comment:
- [ ] This part of the pop-up menu should be aligned in the same vertical line, I think what makes them look as if they are not is the length of the text in each `li` element.

Your design
![image](https://user-images.githubusercontent.com/81932478/180203820-e165efc4-1b9e-40e3-928a-e971644832da.png)

Figma design
![image](https://user-images.githubusercontent.com/81932478/180203991-791ea144-93d5-4b3b-bef4-ef12a4d77292.png)

- [ ] Also it would be nice if you could add a margin-bottom to your design as is in the Figma template.

Your design
![image](https://user-images.githubusercontent.com/81932478/180204336-ae8457f3-7a33-403e-a613-6151324c5752.png)

Figma template
![image](https://user-images.githubusercontent.com/81932478/180204438-62b44492-a323-4a01-b374-cd2a7ee6a415.png)